common/presenters/moves-by-location.js

```diff
@@ -12,7 +12,7 @@ module.exports = function movesByLocation(data, locationKey = 'to_location') {
     locations.push({
       sortKey: location,
       items: sortBy(moves, 'profile.person._fullname').map(
-        moveToCardComponent()
+        moveToCardComponent({ showIdentifierNumber: true })
```

Comment:
Is this still being used? I think what we should be passing in is `locationType` instead, but that will also need to be send in from [the middleware](https://github.com/ministryofjustice/hmpps-book-secure-move-frontend/blob/153c6f50b69528575f790e2829c7ccbf9b7897f4/app/moves/middleware/set-results.moves.js#L5).

The value to use would be `req.location.location_type` I think. And then just pass that all the way down. I know it's a bit fiddly but because this logic needs to be determined based on the location type of the current location and not each move I can't see another way?